### DISCUSSION TITLE
人力终有尽时 | 星光

### DISCUSSION THREAD

**Discussion by giscus** [starguang/myblog#9] [Announcement]
# 人力终有尽时 | 星光

人力终有尽时，明白这句话之前可能是过的最快乐的时候吧，因为不会看到远处那渺茫的未来，只顾着眼下的努力。

http://www.xingguang.host/docs/%E9%9A%8F%E7%AC%94/%E4%BA%BA%E5%8A%9B%E7%BB%88%E6%9C%89%E5%B0%BD%E6%97%B6/

**Comment by IsKan111** [comment]
### 送你一个小心心
